src/fundus/publishers/us/reuters.py

```diff
@@ -59,3 +61,18 @@ def topics(self) -> List[str]:
             # Remove empty topics and duplicates deterministically
             processed_topics = list(dict.fromkeys(topic for topic in topics if topic))
             return processed_topics
+
+    class V1_1(V1):
+        VALID_UNTIL = date.today()
+        _paragraph_selector = XPath("(//div[starts-with(@data-testid, 'paragraph')])[position() > 1]")
```

Comment:
A lot of sports related articles have this tag at the end of the article `--Field Level Media` e.g. https://www.reuters.com/sports/basketball/hot-shooting-suns-wear-down-raptors-2024-03-08/ or https://www.reuters.com/sports/nhl/filip-forsbergs-hat-trick-sends-predators-past-sabres-2024-03-08/